hooks/detect_speculation_and_fake_code.sh

```
@@ -0,0 +1,152 @@
+#!/bin/bash
+# Advanced Speculation & Fake Code Detection Hook for Claude Code
+# Lightweight but comprehensive detection using pattern matching and heuristics
+
+# Hardcoded project root (same as git header approach)
+PROJECT_ROOT="/home/$USER/projects/your-project.com"
```

Shellcheck (0.10.0)

[warning] PROJECT_ROOT appears unused. Verify use (or export if used externally).

(SC2034)